include/FxLine.h

```diff
@@ -75,11 +77,11 @@ class FxLine : public QWidget
 	QColor strokeInnerInactive() const;
 	void setStrokeInnerInactive( const QColor & c );
 
-
 	static const int FxLineHeight;
 
 private:
-	void drawFxLine( QPainter* p, const FxLine *fxLine, const QString& name, bool isActive, bool sendToThis, bool receiveFromThis );
+	void drawFxLine( QPainter* p, const FxLine *fxLine, bool isActive, bool sendToThis, bool receiveFromThis );
+	QString elideName( QString name );
```

Comment:
`const QString & name` should be used instead of `QString name` to avoid the copy of the QString.